example/src/Routes.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { Route, IndexRoute } from 'react-router';
+
+import App from './containers/App';
+import HomePage from './containers/HomePage';
+import ComponentsPage from './containers/ComponentsPage';
+import NotFoundPage from './containers/NotFoundPage';
+
+export default (
+  <Route path="/" component={App}>
+    <IndexRoute component={HomePage} />
+    <Route path="/component/:component" component={ComponentsPage} />
+    <Route path="*" component={NotFoundPage}/> */
```

Comment:
@mkulinski Good catch. I was trying something by commenting code and didnt realize it at the end. 
BTW, typescript was compiling successfully even though for this issue. weird stuff.